README.md

```diff
@@ -45,3 +45,54 @@ As seen in the flowchart previously, the drone ignores the contents of the routi
 ### Nacks
 When sending a Nack, that contains a packet, the packet inside will have its hop index pointing to where the drone would have sent that packet if it wasn't for the nack.
 
+## List of panics
+#### "The Sender<DroneCommand> end of the simulation controller channel unexpectedly got dropped");
+ - There is no link anymore with the simulation controller
+
+#### "There is no connected sender to the drone's receiver channel and no DroneCommand::Crash has been received"
+ - Trigger only if all the channels have been deleted even the controller
+
+#### "Tried to set an invalid pdr value of {pdr}, which is not in range (0.0..=1.0)"
+ - Trigger if you try to set an incorrect new value of the pdr
+
+#### "Cannot add a channel with the same NodeId of this drone (which is {})"
+ - You cannot connect the drone to itself or another drone with the same id
+
+#### "Cannot remove channel to {node_id}: it does not exist"
+ - You try to remove a channel that already doesn't exist
+
+#### "not expecting a packet of type flood request"
+ - Not suppose to be trigger, call of the `process_not_flood_request` function with a flood request
+
```

Comment:
Yeah, as you say given that this is not supposed to be triggered I wouldn't even include it here